projects/Mallard/src/helpers/files.ts

```diff
@@ -3,7 +3,6 @@ import RNFS from 'react-native-fs'
 import { Issue } from 'src/common'
 import { FSPaths } from 'src/paths'
 import { IssueSummary, editions } from '../../../Apps/common/src'
-import { lastNDays } from './issues'
```

Comment:
Is this now being used anywhere? Could this be deleted?

Reply:
It seems to be in use in various places, including in push-tracking to keep last N days logs